panels/history/ha-panel-history.html

```diff
@@ -35,8 +35,12 @@
         on-tap="handleRefreshClick"></paper-icon-button>
 
       <div class$="[[computeContentClasses(narrow)]]">
-        <paper-input label='Showing entries for' id='datePicker'
-          value='[[selectedDate]]'></paper-input>
+        <paper-input
+          label='Showing entries for'
+          id='datePicker'
+          value='[[selectedDateStr]]'
+          on-focus='datepickerFocus'
```

Comment:
I added the on-focus attribute here. It was included in the logbook and the function was already written. I guessed it was just overlooked. Alternatively we can remove the function below if it's not needed.